intro.tex

```diff
@@ -36,20 +36,21 @@ \subsection{Early Science scenarios } \label{ssec:scenarios}
 
 \begin{itemize}
 \item \textbf{Scenario A}:
-The full commissioning plan comprising system optimization and science validation is successfully executed as planned. 
-Rubin Operations then carries out an Operations Rehearsal and Operations Readiness Review (ORR) to effectively conduct a \textit{full dress rehearsal} of science operations and demonstrate the readiness of the Operations team to execute the 10-year survey. 
+Rubin Observatory is ready to execute the LSST at the completion of the construction project. 
+The Operations team would first conduct a \textit{full dress rehearsal} of science operations to demonstrate team readiness to execute the 10-year survey before proceeding to start execution of the LSST.
 Science-grade data collected during the commissioning System Optimization period and subsequent Science Validation Surveys, \S~\ref{sec:commissioning}, is reprocessed to produce the final Data Preview, DP2, which will be released 6 months following the completion of the Science Validation Surveys.
 
 \item \textbf{Scenario B}:
-On-sky time in commissioning is reduced as the construction work  draws to an end, resulting in the SV surveys not being completed prior to the end of the construction phase.
-The Operations team would spend up to 3 months prior to commencing the 10-year LSST survey completing any remaining SV Survey observations.
-As per Scenario A, data collected during commissioning and the SV Surveys is reprocessed to produce DP2 and an Operations Readiness Review carried out to demonstrate readiness to execute the 10-year survey. 
+Prior to commencing the 10-year LSST, the Operations team decides to spend up to a maximum of 2 months collecting more on-sky data to complement and extend the datasets collected during commissioning. 
+This additional data would serve the dual goals of 1) ensuring that we are fully prepared to start the 10-year survey and 2) providing the community with an exquisite early dataset to work with before DR1. 
+
+As per Scenario A, a \textit{full dress rehearsal} of science operations is carried out to demonstrate team readiness and all data collected during commissioning together with the additional data collected prior to starting the LSST is reprocessed to produce DP2.
 \end{itemize}
 
 In both scenarios it is assumed that the Rubin Construction project delivers an integrated system that can capture, transfer and process science-grade data at the time Operations begins.
 Both scenarios will include alert generation of some type, with the major distinction being the relative availability of templates in time, sky position, and filter.
 
-The First Light observations that form the basis of DP1 must be taken and analysed in order to declare construction completeness, meaning that DP1 is  identical in each scenario.
+The First Light observations that form the basis of DP1 must be taken and analysed in order to declare construction completeness, meaning that DP1 is identical in each scenario.
```

Comment:
DP1 data are necessary but not sufficient to declare completeness. Suggest:

The First Light observations that form the basis of DP1 must be taken and analysed in order to support construction completeness, meaning that DP1 is identical in each scenario.